### PR TITLE
Refactor: Add more layouts and refactored side navigation for user page

### PR DESCRIPTION
**Pull request by jeremyhuang035**

### CODE REVIEW

File: src/components/HomeNavigation/HomeNavigation.tsx [hunk 1]
```diff
@@ -7,10 +7,10 @@ const isLoggedIn = true
 
 const HomeNavigation: FC = () => (
   <nav>
-    <div className="flex justify-between mt-7 mx-[120px] text-white">
+    <div className="flex justify-between pt-7 mx-[120px] text-white">
       <div className="flex gap-10 items-center">
         <Link to="/" className="flex items-center gap-2">
-          <img width={32} src="./logo1_1_noText_big.svg" />
+          <img width={32} src="/logo.svg" />
```
Comment (Qiming-Liu) [inline src/components/HomeNavigation/HomeNavigation.tsx:13]: ./logo.svg

File: src/pages/Login/Login.tsx [hunk 2]
```diff
@@ -21,19 +22,19 @@ const Login: FC = () => {
   useEffect(() => {
     if (user) {
       setTimeout(() => {
-        navigate('/')
+        navigate('/account')
       }, 1500)
```
Comment (Qiming-Liu) [inline src/pages/Login/Login.tsx:26]: Why we have a delay here?

File: src/pages/MyReports/components/Content/Content.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ const Content: FC = () => (
         </div>
       </Button>
       <Button className="w-[250px] h-[90px] hover:bg-hover">Browse Templates</Button>
-      <Link to="/my-reports/history">
+      <Link to="/user/my-reports/history">
```
Comment (Qiming-Liu) [inline src/pages/MyReports/components/Content/Content.tsx:25]: I guess this will not working in GitHub pages

**Comment by Qiming-Liu** [comment]
Overall looks great, I'm ok to merge this for now and make sure to fix these issues later as this is an important refactor.

File: src/components/HomeNavigation/HomeNavigation.tsx (hunk 1 above)
Comment (jeremyhuang035) [reply src/components/HomeNavigation/HomeNavigation.tsx]: fixed

File: src/pages/Login/Login.tsx (hunk 2 above)
Comment (jeremyhuang035) [reply src/pages/Login/Login.tsx]: removed delay

File: src/pages/MyReports/components/Content/Content.tsx (hunk 3 above)
Comment (jeremyhuang035) [reply src/pages/MyReports/components/Content/Content.tsx:25]: preview looks ok

File: .env [hunk 4]
```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_BASE_URL=http://localhost:5000
\ No newline at end of file
+REACT_APP_BACKEND_BASE_URL=http://localhost:8000/api/v1
```
Comment (Qiming-Liu) [inline .env:1]: I didn't see there a port changes to the backend

**Review by Qiming-Liu — CHANGES_REQUESTED** [review]

File: .env (hunk 4 above)
Comment (jeremyhuang035) [reply .env:1]: do you mean use .env.local for testing? I'm currently using .env for local testing, 5000 not working for mac, I was thinking to gitignore this .env file, and set up in the production environment variables manually.

**Pull request merged**